scripts/compare_similarity.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
Rename the script `evaluate_similarity`, as it's not doing comparison but just running a model to evaluate it.